src/world/Objects/Units/Players/Player.h

```diff
@@ -1239,6 +1239,8 @@ class SERVER_DECL Player : public Unit
 
         void GiveXP(uint32 xp, const uint64 & guid, bool allowbonus);       /// to stop rest xp being given
         void ModifyBonuses(uint32 type, int32 val, bool apply);
+        DBC::Structures::ScalingStatDistributionEntry const* getScalingStatDistributionFor(ItemProperties const& itemProto) const;
+        DBC::Structures::ScalingStatValuesEntry const* getScalingStatValuesFor(ItemProperties const& itemProto) const;
```

Comment:
Move these new function declarations to MIT license content. Since these are related to items, put them somewhere in between lines 797 and 813.